app/livechat/client/lib/stream/queueManager.js

```diff
@@ -1,14 +1,32 @@
+import { Meteor } from 'meteor/meteor';
+
 import { APIClient } from '../../../../utils/client';
 import { LivechatInquiry } from '../../collections/LivechatInquiry';
 import { inquiryDataStream } from './inquiry';
 import { call } from '../../../../ui-utils/client';
+import { getUserPreference } from '../../../../utils';
+import { CustomSounds } from '../../../../custom-sounds/client/lib/CustomSounds';
 
 const departments = new Set();
 
+const newInquirySound = () => {
+	const userId = Meteor.userId();
+	const audioVolume = getUserPreference(userId, 'notificationsSoundVolume');
+	const newRoomNotification = getUserPreference(userId, 'newRoomNotification');
+	const audioNotificationValue = getUserPreference(userId, 'audioNotifications');
+
+	if (audioNotificationValue === 'all') {
```

Comment:
Why do we need to check this here? In my case, `audioNotificationValue` is equal to `mentions`, but I have never changed that setting, so I didn't get any sound for new Omnichannel rooms.
The same will happens for regular agents, they need to get `newRoomNotification` sound regardless `audioNotificationValue` since that setting is not designed for Omnichannel.

Reply:
I was looking for the example that we have in chat, where the setting is get by Subscription... Omnichannel is not a regular chat, I know, but is not important to consider this general setting as well?

Reply:
I got your point, but IMHO, for Omnichannel I'd check the opposite:
```suggestion
	if (audioNotificationValue !== 'none') {
```

Reply:
Fair enough! Maybe in another time we can discuss better about notification behaviors and best architecture for project, I was thinking about it and how to improve that in omnichannel 😉

Reply:
> Fair enough! Maybe in another time we can discuss better about notification behaviors and best architecture for project, I was thinking about it and how to improve that in omnichannel 😉

Definitely, It'd be great!